gateway/bin/apicast

```diff
@@ -82,7 +82,9 @@ _LUA_
     $bin = $lua_file;
 }
 
-my @resty_args = ();
+my $nginx = $ENV{TEST_NGINX_BINARY} || $ENV{APICAST_OPENRESTY_BINARY} || 'openresty';
```

Comment:
Any reason why `TEST_NGINX_BINARY` has preference over `APICAST_OPENRESTY_BINARY` ?

We are doing the opposite in https://github.com/3scale/apicast/blob/25009f6e3f6029931066aa1c0737d70014a00570/gateway/src/apicast/cli/command/start.lua#L100

Also, it might be worth it to extract the method that selects the binary to a place where the cli and this bin can call it.

Reply:
Good point, will make it the same.